config/app/php-cli.ini

```diff
@@ -1920,26 +1819,60 @@ ldap.max_links = -1
 ; Useful for internal debugging only.
 ;opcache.protect_memory=0
 
-[xdebug]
-xdebug.remote_enable = On
-xdebug.remote_host = 'dockerhost'
-xdebug.remote_port = '9001'
-xdebug.max_nesting_level = 500
+; Allows calling OPcache API functions only from PHP scripts which path is
+; started from specified string. The default "" means no restriction
+;opcache.restrict_api=
+
+; Mapping base of shared memory segments (for Windows only). All the PHP
+; processes have to map shared memory into the same address space. This
+; directive allows to manually fix the "Unable to reattach to base address"
+; errors.
+;opcache.mmap_base=
+
+; Enables and sets the second level cache directory.
+; It should improve performance when SHM memory is full, at server restart or
+; SHM reset. The default "" disables file based caching.
+;opcache.file_cache=
+
+; Enables or disables opcode caching in shared memory.
+;opcache.file_cache_only=0
+
+; Enables or disables checksum validation when script loaded from file cache.
+;opcache.file_cache_consistency_checks=1
+
+; Enables or disables copying of PHP code (text segment) into HUGE PAGES.
+; This should improve performance, but requires appropriate OS configuration.
+;opcache.huge_code_pages=1
 
 [curl]
 ; A default value for the CURLOPT_CAINFO option. This is required to be an
 ; absolute path.
 ;curl.cainfo =
 
-[apc]
-apc.enabled = On
-apc.enable_cli = On
-apc.shm_size = 128M
-apc.ttl = 3600
-apc.user_ttl = 7200
-apc.max_file_size = 1M
-apc.stat = 1
```

Comment:
@violeng you don't want apc module enable for wordpress? if yes, it's fine to remove it like that.